src/controlplushub.ts

```diff
@@ -30,6 +30,10 @@ export class ControlPlusHub extends LPF2Hub {
         );
     }
 
+    protected _currentPort = 0x3b;
+    protected _voltagePort = 0x3c;
+    protected _voltageMaxV = 9.615;
```

Comment:
I copied this value `9615 / 1000` from the existing code (same with `9620 / 1000` for the PUP hub). However the port information data reports `9600 / 1000` in both cases (like the control plus hub). Can you point me to the source of these values (are they coming from an earlier FW)?

Reply:
So I think the 9614/9620 value came around before we knew how to do port information requests (ie. Before Lego released the LWP document). It was determined by someone actually measuring the voltage. So, pretty close!

I'd say the port information request is probably the source of truth though.

Reply:
Well, this is a impressive result indeed. I think in this case I may unify these values to use the same `_voltageMax` to reduce the code size. The difference in values reported will be less than 1% anyway.